index.js

```diff
@@ -202,13 +202,14 @@ export default class ClearvisioAppointmentBooker {
 
     this.store.on('close', () => {
       element.remove();
+      window.removeEventListener('resize');
```

Comment:
Ez így a világ összes resize listenerét leszedi, tehát ha valaki más is rakna rá resize esemény kezelőt azt is leszedtük.